sanic/request.py

```diff
@@ -170,8 +178,8 @@ def remote_addr(self):
             remote_addrs = [
                 addr for addr in [
                     addr.strip() for addr in forwarded_for
-                ] if addr
-            ]
+                    ] if addr
+                ]
```

Comment:
Apparently pep8 isn't weighing in here, but I prefer the earlier spacing in this case, because it closes the multi line list at the same indentation that it was created.